scripts/build.js

```diff
@@ -0,0 +1,110 @@
+const fs = require('fs');
+const path = require('path');
+
+const { ucs2 } = require("punycode"); 
+const SimpleIcons = require('simple-icons');
+const svg2ttf = require('svg2ttf');
+const SVGPath = require('svgpath');
+const ttf2woff = require('ttf2woff');
+const ttf2woff2 = require('ttf2woff2');
+
+
+const DISTDIR = path.resolve(__dirname, '..', 'font');
+
+const buildSimpleIconsSvgFontFile = () => {
+  let fileContent = '<?xml version="1.0" standalone="no"?>\n'
+                  + '<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">\n'
+                  + '<svg xmlns="http://www.w3.org/2000/svg">\n'
+                  + '<metadata>CC0 1.0 Universal | simple-icons contributors</metadata>\n'
+                  + '<defs>\n'
+                  + '<font id="Simple Icons" horiz-adv-x="1200">\n'
+                  + '<font-face font-family="Simple Icons" units-per-em="1200" ascent="-1" descent="1200" />\n'
+                  + '<missing-glyph horiz-adv-x="0" />\n';
+  
+  let startUnicode = 0xea01;
+  let usedUnicodes = [];
+  
+  let unicodeHexBySlug = {}
+  
+  for (let iconTitle in SimpleIcons) {
+    let unicode = ucs2.decode(
+      String.fromCodePoint(startUnicode++)).map(
+        point => '&#x' + point.toString(16).toUpperCase() + ';')[0];
+    if (usedUnicodes.includes(unicode)) {
+      throw Error(`Unicodes must be unique. Found '${unicode}' repeated`)
+    }
+    
+    let icon = SimpleIcons[iconTitle];
+    const verticalTransformedPath = SVGPath(icon.path).scale(50, -50).toString();
+    
+    fileContent += `<glyph glyph-name="${icon.slug}" unicode="${unicode}" d="${verticalTransformedPath}" horiz-adv-x="1200" />\n`;
```

Comment:
Are the newlines necessary? Not sure if it would reduce the font-size if we remove them but that could be worth considering.

```suggestion
    fileContent += `<glyph glyph-name="${icon.slug}" unicode="${unicode}" d="${verticalTransformedPath}" horiz-adv-x="1200" />`;
```

Reply:
I assumed that we would minify at launch, but there is not much to see inside so I could build it already minified, but also the `<metadata>`, `<defs>`, `<font>`... part?

Reply:
> I assumed that we would minify at launch

Fair enough, I would agree. But, actually, do we need to include `SimpleIcons.svg` when publishing to NPM?


> but also the `<metadata>`, `<defs>`, `<font>` ... part?

I would say not, but I'm not sure. The main reason I asked it here is because it serves no purpose in the code. With the other part at least it makes it a bit more readable than a single long line...

We could consider doing [something similar to the main repo](https://github.com/simple-icons/simple-icons/blob/develop/scripts/templates/icon-object.js) where we have a template file for the `SimpleIcons.svg` file and just insert the long string with all `<glyph>`s at the end. What do you think about that?

Reply:
> But, actually, do we need to include SimpleIcons.svg when publishing to NPM?

FontAwesome does that: `.svg`, `.eot`, `.ttf`, `.woff` and `.woff2`. I suppose that could be useful to someone.

> We could consider doing something similar to the main repo where we have a template file for the SimpleIcons.svg file and just insert the long string with all <glyph>s at the end. What do you think about that?

Yes, good idea. I will do that removing all newlines after glyphs, keeping the metadata readable :+1:

Reply:
Done :wrench: :sparkles:

Reply:
> FontAwesome does that: .svg, .eot, .ttf, .woff and .woff2. I suppose that could be useful to someone.

Hmm, still not 100% convinced but this doesn't need to block this PR from being merged.

I did some local testing and it seems each font works fine without the `.svg` there. Given that the SVGs are already available through the main package I feel like it is a bit redundant to include the SVG file here as well :thinking: It's not much (1.6MB) but I think we should consider adding the following to `.npmignore`:

```
/font/SimpleIcons.svg
```

If not here, I will open an issue for it so that we can decide on it later.

Reply:
OK, I will do and, if someones need, could be discussed in aonther issue.

Reply:
Done! :wrench: